src/app/footer-navbar/footer-navbar.component.html

```diff
@@ -1,12 +1,20 @@
-<footer>
-  <div class="left-side">
-      <a routerLink="/about" routerLinkActive="active">About</a>
-      <a href="//blog.fossasia.org">Blogs</a>
-      <a href="//github.com/fossasia/susper.com">Code</a>
+<nav class="navbar navbar-default footer navbar-fixed-bottom" role="navigation">
+  <div class="navbar-header">
+    <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
+      <span class="icon-bar"></span>
+      <span class="icon-bar"></span>
```

Comment:
https://github.com/nikhilrayaprolu/susper.com-1/commit/4d5529497eda6f5ae6ef314d663ada257458fd86#diff-1f7630f00b341e8ec72a4cdf9a23288bR4 that's the past footer we have(made a month back by @isuruAb ) , just same lines with fixed lines. that was removed in #211 .which I fixes here.